packages/rust_analyzer/project.bri

```diff
@@ -0,0 +1,47 @@
+import * as std from "std";
+import { cargoBuild } from "rust";
+
+export const project = {
+  name: "rust_analyzer",
+  version: "2025-08-25",
+  repository: "https://github.com/rust-lang/rust-analyzer.git",
+};
+
+const source = Brioche.gitCheckout({
+  repository: project.repository,
+  ref: project.version,
+});
+
+export default function rustAnalyzer(): std.Recipe<std.Directory> {
+  return cargoBuild({
+    source,
+    path: "crates/rust-analyzer",
+    env: {
+      CFG_RELEASE: project.version,
+    },
+    runnable: "bin/rust-analyzer",
+  });
+}
+
+export async function test(): Promise<std.Recipe<std.File>> {
+  const script = std.runBash`
+    rust-analyzer --version | tee "$BRIOCHE_OUTPUT"
+  `
+    .dependencies(rustAnalyzer)
+    .toFile();
+
+  const result = (await script.read()).trim();
+
+  // Check that the result contains the expected version
+  const expected = `rust-analyzer ${project.version}`;
+  std.assert(result === expected, `expected '${expected}', got '${result}'`);
+
+  return script;
+}
+
+export function liveUpdate(): std.Recipe<std.Directory> {
+  return std.liveUpdateFromGithubReleases({
+    project,
+    matchTag: /^(?<version>.+)$/,
```

Comment:
I override the matchTag since the pattern is not a server like version, it is a date.